pulpcore/openapi/__init__.py

```diff
@@ -389,7 +389,7 @@ def parse(self, request, public):
                 continue
 
             # Removes html tags from OpenAPI schema
-            if "include_html" not in request.query_params:
+            if request and "include_html" not in request.query_params:
```

Comment:
As "include_html" needs to be added, i'd like this to default to strip the tags:
```suggestion
            if not (request or "include_html" in request.query_params):
```